### PR TITLE
pushing to git

### PR DESCRIPTION
**Pull request by milabakhmat**

### CODE REVIEW

File: target/site/serenity/photo-linux-firefox-119.0-alice-ensures-that-the-text-of-page-element-326383c13a.png
Comment (ovartem) [inline target/site/serenity/photo-linux-firefox-119.0-alice-ensures-that-the-text-of-page-element-326383c13a.png:1]: All this folder needs to be:
1. deleted
2. changes with the deletion should be commited
3. folder added to gitignore file

File: test/serenity/github-api/GitHubStatus.js
Comment (ovartem) [inline test/serenity/github-api/GitHubStatus.js:1]: test folder should be deleted, committed, and added to gitignore as well. (same as target folder)

File: Unit_9/test/test1.spec.js [hunk 1]
```diff
@@ -0,0 +1,24 @@
+const { describe, it } = require("mocha");
```
Comment (ovartem) [inline Unit_9/test/test1.spec.js:1]: looks like you have not committed all the necessary files. Could you please check that

File: serenity.properties
Comment (ovartem) [inline serenity.properties:1]: Where did you get serenity?

**Review by ovartem — CHANGES_REQUESTED** [review]
Well. Looks like you have made a lot. Still we need to structure the code and your understanding. Let's have a quick call and you'll show me where did you get serenity and all other code

File: .vscode/launch.json [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+    "configurations": [
+      {
+        "type": "node",
+        "request": "launch",
+        "name": "Launch Program",
+        "skipFiles": ["<node_internals>/**"],
+        "program": "${file}"
+      },
+      {
+        "name": "run select spec",
```
Comment (ovartem) [inline .vscode/launch.json:15]: Ооо! це круто!

**Review by ovartem — CHANGES_REQUESTED** [review]
Well done!
Much better now.
Let's continue working on improvements

File: wdio.conf.js [hunk 3]
```diff
@@ -21,9 +21,9 @@ exports.config = {
   // then the current working directory is where your `package.json` resides, so `wdio`
   // will be called from there.
   //one file
-  specs: ["./Unit_9/test/test2.spec.js"],
-//folder with tests
- // specs: ["./Unit_9/test/*.js"],
+  specs: ["./test/specs/suite1.spec.js"],
```
Comment (hlazkov) [inline wdio.conf.js:24]: We'd better to remove example from the folder and use something like ./test/specs/*.spec.js here

File: test/serenity/github-api/GitHubStatus.js
Comment (hlazkov) [reply test/serenity/github-api/GitHubStatus.js:1]: Same with test reports!

File: pageObject/loginPage.js [hunk 4]
```diff
@@ -13,10 +13,8 @@ class LoginPage extends Page {
         await this.loginBtn.click();
     }
 
-    async openSite() {
-        await browser.url(
-          "https://www.saucedemo.com/"
-        );
+    open() {
+        super.open(" ");
```
Comment (hlazkov) [inline pageObject/loginPage.js:17]: Do we need space here? looks strange

File: test/serenity/github-api/GitHubStatus.js
Comment (hlazkov) [reply test/serenity/github-api/GitHubStatus.js:1]: Oh, its done, sry!

File: test/specs/suite1.spec.js [hunk 5]
```diff
@@ -0,0 +1,49 @@
...
+    //display it in the cart and compare with the Inv page
+    await inventoryPage.openCart();
+
+    const itemInCart = shoppingCart.itemsInCart[0].getText();
+    await expect(itemInCart).toEqual(addedItem);
+
+    //remove the backpack and check that the cart is empty
+    await shoppingCart.removeBtns[0].click();
+
+    await shoppingCart.removedItem.isExisting();
```
Comment (hlazkov) [inline test/specs/suite1.spec.js:41]: What does this method do? oO

File: test/specs/test2.spec.js [hunk 6]
```diff
@@ -10,33 +10,33 @@ const inventoryPage = require("../../pageObject/inventoryPage");
...
     inventoryPage.shoppingCartBadge.isEqual(1);
   });
 
   it("should display the chosen item in the shopping cart and compare the added item with the Inv page", async () => {
     await inventoryPage.openCart();
 
-    const itemInCart = shoppingCart.removeBackpackBtn.getText();
-    await expect(itemInCart).toEqual(inventoryPage.removeBackpackBtn.getText());
+    const itemInCart = shoppingCart.removeBtns[0].getText();
+    await expect(itemInCart).toEqual(inventoryPage.removeBtns[0].getText());
```
[the remark this answers is not in the document]
Comment (milabakhmat) [reply test/specs/test2.spec.js]: thanks! const on inv page is added

File: pageObject/loginPage.js (hunk 4 above)
Comment (milabakhmat) [reply pageObject/loginPage.js:17]: thanks! no omre space

open() {
    super.open("");
  }

File: test/specs/suite1.spec.js (hunk 5 above)
Comment (milabakhmat) [reply test/specs/suite1.spec.js:41]: checks that no item is in the shopping cart )

**Review by hlazkov — APPROVED** [review]
Nice!

**Pull request merged**